docs/source/standard.rst

```diff
@@ -0,0 +1,55 @@
+Card Game Notation Standard
+===========================
+This standard uses TOML_ as notation language.
+
+Header
+-------
+All meta data is defined in the Header.
+This contains information about teams and players.
+Furthermore, the location and datetime is stated.
+
+.. code-block:: toml
+
+    [header]
+    # Timestamp of the starting time in RFC 3339
+    timestamp = 2022-08-06T11:22:00
+    # Location of the game or the server url
+    location = "Hamburg"
+    # Unsigned Integer
+    number_of_teams = 2
+    [[players.seat_1]]
+    # Unsigned Integer: ID of team starting with 0
+    team = 0
+    name = "John"
+    [[players.seat_2]]
+    team = 1
+    Name = "Lucy"
+
+Stack
+-----
+The data contains one mandatory field ``stack`` that is a list of strings representing a card.
+Furthermore it has an optional field ``hands`` that is a table of the players' cards in hand.
+One card can be represented with
+
+.. code-block:: toml
+
+    suit = "[♦♣♥♠]"
+    rank = "[2-10JQKA]"
+
+.. code-block:: toml
+
+
+    [[stack.card]]
+    suit="heart"
+    rank="A"
+    [[data.stack.card]]
+    suit="heart"
+    rank="K"
+
+    [[hands.player_1]]
+    [[hands.player_1.card]]
+    suit="heart"
+    rank="2"
```

Comment:
this toml is a little bit weird.
a table uses `[...]` while a table in an array uses `[[...]]`.

Reply:
check out this and the following paragraphs about tables and arrays of tables: https://toml.io/en/v1.0.0#array

Reply:
I was also thinking, that actually don't need the `hands`, right?